### PR TITLE
Phas2-HGX350B Attempt to make a flat silicon file for HGNose descriptiopn - first step to make the next version of HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Attempt to make a flat silicon file for HGNose descriptiopn - first step to make the next version of HFNose

#### PR validation:

The flat file is created. The next step will be to use this file to make a new version of HGNose along with the V18 version of HGCal

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild please test

File: Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc [hunk 1]
```diff
@@ -26,7 +26,7 @@
 #include <unordered_set>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
[the remark this answers is not in the document]
Comment (srimanob) [reply Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc:29]: Can we completely remove this line? As every times you want to debug, you need to update the code. But with `USER_CXXFLAGS="-DEDM_ML_DEBUG" scram b -j8`, it should do the same without the need to update the code (checkout and scram b).

See https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger#LogDebug

**Comment by bsunanda** [comment]
@cmsbuild Plese test

**Comment by bsunanda** [comment]
@cmsbuild please test

**Comment by bsunanda** [comment]
+geometry

**Comment by bsunanda** [comment]
@srimanob Please approve this

**Comment by srimanob** [comment]
+Upgrade

**Comment by rappoccio** [comment]
+1

**Pull request merged**